vivo-project/VIVO
# [VIVO-1996] Retain language tags in manage publications query results to allow for language filtering

**[https://jira.lyrasis.org/browse/VIVO-1996](https://jira.lyrasis.org/browse/VIVO-1996)**:

# How should this be tested?
Add or visit a publication with title (label) in multiple languages.
From an author's profile, click manage publications.  Note that the publication is repeated for each of its titles.
Install PR:
Visit manage publications and note that the publication appears once with language-appropriate title.

# Interested parties
@VIVO-project/vivo-committers
